ouroboros-consensus-byron/src/Ouroboros/Consensus/Byron/Ledger/Forge.hs

```diff
@@ -139,10 +147,12 @@ forgeRegularBlock cfg bno sno st txs isLeader =
     epochSlots :: CC.Slot.EpochSlots
     epochSlots = byronEpochSlots cfg
 
+    computedMaxTxCapacity = computeMaxTxCapacity st maxTxCapacityOverride
+
     blockPayloads :: BlockPayloads
     blockPayloads =
         foldr
```

Comment:
It occurs to me that `foldr` was fine when we were preserving the total length of the list, but `foldr` does not work now that we're dropping some of the list. Specifically, the current logic (5a1d48e) takes the longest _suffix_ of the input list that contains less than some total number of bytes.